util/start_test

```diff
@@ -1099,6 +1099,12 @@ def set_up_executables():
                         "{0}".format(prediff))
                 finish()
             chpl_system_prediff.append(prediff)
+    if launcher == "slurm-srun":
+        # With Slurm-based launcher, auto-run prediff-for-slurm-srun.
+        pdSlurm = os.path.join(util_dir, "test", "prediff-for-slurm-srun")
```

Comment:
`pdSlurm` reads odd to me. I'd probably use `slurm_prediff` or `prediff_for_slurm`